src/index.js

```diff
@@ -0,0 +1,8 @@
+var checkWord = require('check-if-word')
+words = checkWord('en')
+
+const checkIfWord = function (word) {
```

Comment:
Looks good.

As an architectural decision, are we keeping all logic functions inside index.js?

Reply:
On second check, this library seems to not be particularly performant?
![image](https://github.com/user-attachments/assets/9b07373e-28ec-4b70-b43b-366b78c09a1c)

Reply:
`check-word` https://www.npmjs.com/package/check-word seems to be much more performant on testing.
![image](https://github.com/user-attachments/assets/69310ab7-c679-410a-9ddd-983a221461c6)

Reply:
I honestly thought so too haha. Thanks for providing an alternative.

Reply:
> Looks good.
> 
> As an architectural decision, are we keeping all logic functions inside index.js?

I suppose this is something we can work on on a later date.